### DISCUSSION TITLE
How to add a fallback for custom/arbitrary values?

### DISCUSSION THREAD

**Discussion by pavelbelyaev98** [tailwindlabs/tailwindcss#10867] [Help] [answered]
Hello,

I'm currently working on a project where I need the ability to define custom properties for styles, along with specifying a second custom property as a fallback. I'm trying to achieve something similar to the following example:

```
bg-[var(--my-var, var(--my-background, pink))]
```

I found that this concept is [well-documented in the Mozilla Developer Network (MDN)](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#custom_property_fallback_values), but I am unsure of how to effectively implement this within the context of TailwindCSS.

Thank you in advance for your assistance!

**Comment by wongjn** [comment, marked as answer]
Remove the spaces (since spaces separate class names) and it should work: `bg-[var(--my-var,var(--my-background,pink))]`.

**Reply by pavelbelyaev98** [reply]
That was it! Thanks.
